packages/php-wasm/node/src/test/rotated-php.spec.ts

```diff
@@ -0,0 +1,128 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { NodePHP } from '..';
+import { LatestSupportedPHPVersion, rotatedPHP } from '@php-wasm/universal';
+
+const recreateRuntime = async (version: any = LatestSupportedPHPVersion) =>
+	await NodePHP.loadRuntime(version);
+
+async function rotate(php: any) {
+	await php.run({
+		code: `<?php echo 'hi';`,
+	});
+	await php.run({
+		code: `<?php echo 'hi';`,
+	});
```

Comment:
what's the point of these lines?

Reply:
These `run()` calls are meant to trigger the internal PHP runtime rotation. I just replaced the `rotate()` function with something hopefully more readable:

```
		// Rotate the PHP runtime
		await php.run({ code: `` });
```

Note there's no "echo" involved